wdl/model/draft2/src/main/scala/wdl/draft2/model/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -301,6 +305,8 @@ trait WdlStandardLibraryFunctions extends WdlFunctions[WomValue] {
 }
 
 object WdlStandardLibraryFunctions {
+  private val logger = LoggerFactory.getLogger(WdlStandardLibraryFunctions.getClass)
```

Comment:
Is this better that mixing in `with StrictLogging`?

Reply:
I got some cryptic compiler error when trying to use scala-logging pointing to this previous `import` line:

```
/Users/kshakir/src/cromwell/wdl/model/draft2/src/main/scala/wdl/draft2/model/expression/WdlStandardLibraryFunctions.scala:12:36
encountered unrecoverable cycle resolving import.
Note: this is often due in part to a class depending on a definition nested within its companion.
If applicable, you may wish to try moving some members into another object.
import wdl.draft2.model.expression.WdlStandardLibraryFunctions.{crossProduct => stdLibCrossProduct, _}
```

Don't feel like figuring it for now.